projects/kit/components/input/input.component.ts

```diff
@@ -117,6 +119,16 @@ export class TuiInputComponent
         this.open = false;
     }
 
+    override writeValue(value: string | null): void {
```

Comment:
Why `null` ?

Empty text field - empty string, is it ?

Reply:
fixed